test/runTests.ts

```diff
@@ -18,11 +26,21 @@ async function main(): Promise<void> {
         // Passed to --extensionTestsPath
         const extensionTestsPath = path.resolve(__dirname, "./index");
 
+        // Open VSCode with the examples folder, so any UI testing can run against the examples.
+        // Also install the c# extension which is needed for hybrid binary module debug testing
```

Comment:
Where does that install happen? And what happened to `--disable-extensions`?

Reply:
Ok so I found where it gets installed, and so I think we should still start with users' extensions disabled, otherwise things get messy!

Reply:
Also it eliminates the problem where you had to run stable to test insiders, etc., you can now test any vscode version so we can run both stable and insiders tests, run against a specific vscode version, etc.

Reply:
441db78bc6c314895c560048e7366337f708b78b

Reply:
Got it, awesome!